pkgs/tools/security/passage/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, installShellFiles, asciidoctor, xorg, python3, passAlias ? false }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "passage";
+  version = "0.1.0";
+
+  src = builtins.fetchGit {
```

Comment:
Please use fetchFromSourcehut instead.

Reply:
Fixed it, now it uses fetchFromSourcehut.